roofit/roofitcore/inc/RooGlobalFunc.h

```diff
@@ -201,6 +201,8 @@ RooCmdArg EventRange(Int_t nStart, Int_t nStop) ;
 RooCmdArg Extended(bool flag=true) ;
 RooCmdArg DataError(Int_t) ;
 RooCmdArg NumCPU(Int_t nCPU, Int_t interleave=0) ;
+RooCmdArg Parallelize(Int_t nWorkers, bool parallel_gradient, bool parallel_likelihood) ;
+RooCmdArg NewStyle(bool flag=false) ;
```

Comment:
```suggestion
RooCmdArg Parallelize(int nWorkers, bool parallelGradient, bool parallelLikelihood) ;
RooCmdArg NewStyle(bool flag=false) ;
```
* no need use `*_t` types if not for things written to files
* RooFit uses `camelCase`